src/style.css

```diff
@@ -0,0 +1,120 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.goods-card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+
+  font-family: Roboto, sans-serif;
+}
+
+.goods-card__image {
+  margin: 0 3px 40px;
+}
```

Comment:
try to set fixed height for test pass

Reply:
In video mentor advices don't set fixed values to height or witdth because if we have content bigger than container it will come out from container, but okay